### ISSUE TITLE
Adding/Replacing existing kcContext mock data

### ISSUE THREAD

**Issue by cbude** [InseeFrLab/keycloakify#21]
I tried to add required actions to the info.ftl mock but it throws an error when I try to get the mocked context as shown in the image below. Have I made a mistake or does the feature not work exactly as intended? 

The field is supposed to be undefined and according to the code in deepassign it should just use the new provided property.

```
 "mockData": [
    {
      "pageId": "info.ftl",
      requiredActions : [
        "CONFIGURE_TOTP",
        "UPDATE_PASSWORD",
        "UPDATE_PROFILE",
        "VERIFY_EMAIL",
        "terms_and_conditions"
      ]
    }
]
```
![image](https://user-images.githubusercontent.com/24910948/125673095-d35f9901-4415-44a7-8067-4378d83092b6.png)

**Comment by garronej** [comment]
Hi,
At first sight it looks like an error on my end.
I will look into it.

**Comment by garronej** [comment]
Confirmed error on my side... fixing

**Comment by garronej** [comment]
Should be fixed 2.0.9, let me know if you have any issues

**Comment by cbude** [comment]
Works perfectly! Thanks for the help 😄
